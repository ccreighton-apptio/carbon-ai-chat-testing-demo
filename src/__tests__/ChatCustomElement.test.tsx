import React from 'react';
import { render } from '@testing-library/react';
import { ChatCustomElement } from '@carbon/ai-chat';

describe('ChatCustomElement', () => {
  it('should match snapshot', () => {
    const { container } = render(
      <ChatCustomElement
        config={{}}
        className="custom-chat-element"
        data-testid="chat-custom-element"
      />
    );
    
    expect(container.firstChild).toMatchSnapshot();
  });
});