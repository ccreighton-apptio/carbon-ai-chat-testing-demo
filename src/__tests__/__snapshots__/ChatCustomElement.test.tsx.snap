// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`ChatCustomElement should match snapshot 1`] = `
<div
  class="custom-chat-element"
>
  <cds-aichat-react />
</div>
`;
